k8s/operator/templates/rbac.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ include "sdp-operator.fullname" . }}-role
+  name: {{ include "sdp-operator.fullname" . }}-r
```

Comment:
Why the -r suffix?

Reply:
to make clear it was a role, that's what the rolebinding is referencing. We can improve naming later if we want. I dont think we have more roles there so the suffix is not strictly needed.